### ISSUE TITLE
Customizing Sampling Behavior

### ISSUE THREAD

**Issue by Ja1Zhou** [sgl-project/sglang#89]
Hi, is there an interface to specify logits processors as in vLLM? 

If possible, could you specify how we can customize the sampling behavior during generation?

**Comment by Ja1Zhou** [comment]
Closing this issue for now
